<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundColor</class>
 <widget class="QDialog" name="BackgroundColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Background Color</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonTop">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Top Color</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonBottom">
         <property name="text">
          <string>Bottom Color</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGrad">
         <property name="text">
          <string>Gradient</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxGrad</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButtonBottom</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
